### ISSUE TITLE
Can I dynamically adjust the learning rate for my training process?

### ISSUE THREAD

**Issue by jaylongnwnr** [ultralytics/yolov5#11607]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello, I now see that my loss can't be lowered during the training process, so I want to improve this problem by adjusting the learning rate. I thought I could use --lr to set my parameters, but later found this doesn't work, is there any more direct way to use?

Thanks

**Comment by glenn-jocher** [comment]
Hello @jaylongnwnr, thanks for using YOLOv5 and posting your question. Yes, you can dynamically adjust the learning rate during training using the `--lr` flag. However, it is not the only way to control the learning rate. You can also use a scheduler object with various available options such as [OneCycleLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html) or [CosineAnnealingLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.CosineAnnealingLR.html). These schedulers can be defined in train.py and automatically adjust the learning rate as your model trains, based on factors such as the number of epochs completed or the loss value.

Thanks for your support!
